Mathlib/GroupTheory/CommutingProbability.lean

```diff
@@ -41,6 +41,31 @@ theorem commProb_def :
   rfl
 #align comm_prob_def commProb_def
 
+theorem commProb_prod (M' : Type _) [Mul M'] : commProb (M × M') = commProb M * commProb M' := by
```

Comment:
Some easy similar results if you care:

* `commProb (MulOpposite M)`
* `commProb (ULift M)`